.github/workflows/blade-old-release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Blade Old
+
+on: 
+  push:
+    branches: [master]
+
+env:
+  GITHUB_ACCESS_TOKEN: ${{ secrets.CI_BOT_TOKEN }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Default is `1` which should be fine. Why `0`? Is it even a valid value? 🤔

Reply:
Yes it's a valid option. [Source](https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches)

changeset needs this. Right from the documentation of changeset
> This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits